controllers/OrganisasiController.js

```
@@ -1,44 +1,60 @@
+      periode_dimulai,
+    } = organisasi;
+    val =
+      typeof nama === "string" &&
+      Array.isArray(kode_divisi) &&
+      Array.isArray(divisi) &&
+      Array.isArray(kode_jenis_surat) &&
+      Array.isArray(jenis_surat) &&
+      typeof format_nomor_surat === "string" &&
+      typeof periode === "number" &&
+      typeof password === "string" &&
+      typeof periode_dimulai === Date;
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)